### PR TITLE
V1 in maintenance

### PR DESCRIPTION
**Pull request by dholbach**
Thanks @hiddeco and @mewzherder for your work in #22 - combining the two here.

Closes: #18

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -11,6 +11,10 @@ slack                = "https://cloud-native.slack.com/messages/flux"
 mailing_list         = "https://lists.cncf.io/g/cncf-flux-dev"
 font_awesome_version = "5.3.1"
 
+notification = """
+📣 **[Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)!** We will soon start publishing guides which help you migrate Flux v2 and the GitOps Toolkit. **[Learn more](https://toolkit.fluxcd.io/)** and **[join the discussion!](https://github.com/fluxcd/toolkit/discussions)**
```
Comment (hiddeco) [inline config.toml:15]:
```suggestion
📣 **[Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)!** We will soon start publishing guides which help you migrate to Flux v2 and the GitOps Toolkit. **[Learn more](https://toolkit.fluxcd.io/)** and **[join the discussion!](https://github.com/fluxcd/toolkit/discussions)**
```

File: layouts/partials/notification.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{{ $notification := site.Params.notification | markdownify }}
+<div class="notification is-warning mb-0">
+  {{ $notification }}
+</div>
```
Comment (hiddeco) [inline layouts/partials/notification.html:4]:
```suggestion
</div>
```

Comment (dholbach) [reply layouts/partials/notification.html:4]: Hum...?

**Review by hiddeco — APPROVED** [review]
:rocket:

**Comment by dholbach** [comment]
:shipit:

**Pull request merged**